src/main/java/org/myrobotlab/service/Pcf8574.java

```diff
@@ -575,6 +563,34 @@ int readRegister() {
     return ((int) readbuffer[0]) & 0xff;
   }
 
+  /**
+   * This returns the last set value on the pin.
+   * When this is set to True, a read value of false indicates the pin is pulled low by external influence.
+   * 
+   * @param address Integer
+   * The pin to be looked at
+   * @return current state of the output register for the pin
+   */
+  public boolean readOutputPin(int address) {
+    int value = (writeRegister >> address) & 1;

```

Comment:
I think it should return an int .. it would be really good if it would match an interface.
Mrl is too Arduino specific.  In the big picture this should follow file reading interface although imo Java's not the best spec for this.
perhaps 
```java
public int read(int address) {
```